ParsedownExtra.php

```diff
@@ -441,10 +461,16 @@ protected function parseAttributeData($attributeString)
             {
                 $Data['id'] = substr($attribute, 1);
             }
-            else # "."
+            elseif ( $attribute[0] === '.' )
             {
                 $classes []= substr($attribute, 1);
             }
+            elseif ( strpos($attribute, '=') )
+            {
+                preg_match('#([\w-]+)="([\D\w-]+)"#', $attribute, $match);
```

Comment:
Can you please make the `""` optional? This makes it compatible to [PHP Markdown Extra's documentation](https://michelf.ca/projects/php-markdown/extra/#spe-attr) - even it isn't implemented by PHP Markdown Extra...